web/docs/setup/README.md

```diff
@@ -63,25 +63,25 @@ If you have a Linux at your fingertips, just download and extract our package
 with a static binary:
 
 ```bash
-mkdir /opt/vast
-cd /opt/vast
-wget https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz
-mkdir -p /opt/vast
-tar xzf vast-linux-static.tar.gz -C /opt/vast
-export PATH="/opt/vast/bin:$PATH" # based on your shell, e.g., fish_add_path /opt/vast/bin
-vast start
+mkdir /opt/tenzir
+cd /opt/tenzir
+wget https://github.com/tenzir/tenzir/releases/latest/download/tenzir-linux-static.tar.gz
+mkdir -p /opt/tenzir
+tar xzf tenzir-linux-static.tar.gz -C /opt/tenzir
+export PATH="/opt/tenzir/bin:$PATH" # based on your shell, e.g., fish_add_path /opt/tenzir/bin
+tenzir start
 ```
 </TabItem>
 <TabItem value="docker" label="Docker">
 For a container deployment, use our official Docker image:
 
 ```bash
-docker pull tenzir/vast
+docker pull tenzir/tenzir
 mkdir -p /tmp/db # persistent state
-docker run -dt --name=vast --rm -p 5158:5158 -v /tmp/db:/var/lib/vast \
-  tenzir/vast start
+docker run -dt --name=tenzir --rm -p 5158:5158 -v /tmp/db:/var/lib/tenzir \
+  tenzir/tenzir start
```

Comment:
This must be adapted. I'm not yet sure how we want to model the commands for this; we may end up either not specifying a default command, making the default command tenzirctl, or introducing separate images for the three binaries. Also cc @tobim, because we haven't talked about the entrypoint for the Docker images so far.

Reply:
Also CC @rdettai regarding how to structure the Docker images.

The DX might be best with a single Docker image, but I'm not sure about the UX.

Reply:
[Linking the description of the various binary types](https://github.com/tenzir/vast/pull/3187) in case I'm not the only one who missed this. I might have missed the core rational for adding `tenzirctl` and `tenzird`. Is it just to emphasize the 2 main functionalities of tenzir?

I definitively wouldn't go as far as creating 3 images. You can always use another binary with `--entrypoint` (I think you mentioned Clickhouse as an example at some point and that's what they do `docker run -it --rm --link some-clickhouse-server:clickhouse-server --entrypoint clickhouse-client clickhouse/clickhouse-server`).

From a UX perspective, do we want to promote `exec` (and hence `tenzir`) as a prime feature of the product? If that's the case, it should be the entrypoint. That would be the command our users have most "direct" interactions with. For instance when setting up a node with the SaaS product, the `tenzird` functionality will already be configured in the compose our users download from the config page, so they won't even see the extra `--entrypoint`.

Reply:
Thanks for sharing the perspective. We'll continue the discussion in Slack.